governance.md

```diff
@@ -0,0 +1,175 @@
+  Association, based on the inputs of the US-RSE Association membership.
+* The Steering Committee is ultimately responsible for day-to-day activities, operations, and
+  development, but many of these actions and activities will be initiated, organized, and managed by
+  community committees and members at large.
+* Members of the Steering Committee have the ability to grant or remove administrative rights for
+  US-RSE Association umbrella technologies including but not limited to:
+    * All repositories in the US-RSE GitHub organization
+    * All mailboxes and mailing lists associated with the us-rse.org domain name
+    * The us-rse.org domain and website
+    * The US-RSE slack channel
+    * Any other official accounts intended to represent the overall organization, now or in the
+      future (e.g., Twitter, Facebook, LinkedIn, etc.)
+
+The Steering Committee strives for consensus in its decisions. In the case of a dispute or argument
+within the community, a majority vote of all of the Steering Committee members will result in a
+final decision. In the event of a tie, the Steering Committee chair will cast the deciding vote.  At
+all Committee meetings, quorum is at least one half of the members.
+
+### Steering Committee Elections and Terms
+
+After the conclusion of the founding Steering Committee's tenure, all future Steering Committee
+members will be elected by the general US-RSE membership.  Steering committee members are elected to
+a 2-year term. To provide continuity, steering committee members will have staggered terms, with
+four or five members elected each year.
+
+Nominations for open positions will be solicited from all members at least 6 weeks before the
+election.  The list of nominated candidates will be announced at least 3 weeks before the election.
+All US-RSE members in good standing on the date that the nominations are opened are eligible to be
+nominated as candidates.
+
+### Steering Committee Meetings and Communication
+
+Steering Committee meetings will take place at a minimum of once per quarter, and typically will be
+held monthly. All Steering Committee meetings will have notes recorded and made available to the
+US-RSE Association members. The tallies of any votes of the Steering Committee will also be made
+available.
+
+### Steering Committee Officers
+
+Officers consist of a Chair, a Vice-Chair, Election Chair, and a Secretary, elected by the Steering
+Committee. Officer elections will be held at the meeting following the election of new members to
+the Steering Committee.
+
+#### Chair
+
+The Chair will set the agenda and preside over Steering Committee meetings or arrange for the Vice
+Chair or another Steering Committee member to do so.
+
+#### Vice-Chair
+
+In the event of the absence of the Chair, the Vice-Chair will preside at meetings. The Vice-Chair
+will be responsible for ensuring that the bi-monthly newsletter will be sent out.  In addition, the
```

Comment:
I'm not sure that we should specify bi-monthly, as at some point, we might want this to be monthly or even weekly.  Maybe we say "that a newsletters is sent out at least bi-monthly" or something even more general?

Reply:
maybe that a newsletter will be sent out at least 3? times per year

Reply:
or just "will be sent out regularly"?

Reply:
would once every 2 years be ok?  That would be regular.

Reply:
For a governance document, I'm not sure we even need to specify newsletter responsibility. Is it critical to include? It could very easily become someone else's responsibility, a community coordinator/manager or committee. All that said, if it says in, I'd be fine with some form of regular rather than bi-monthly.

Reply:
I agree with Ian, I don't think that needs to specified in a governance document. This iteration of the Steering Committee likes doing it bi-monthly, later iterations may choose monthly, or semi-annual, or even just to eliminate it in favor of more frequent blog/twitter/tiktok/whatever posts. I'm in favor of removing this entirely or if we must, include something like: "The Vice-Chair will be responsible for ensuring the community is kept informed of Association activities through regular newsletters, social media posts, etc."

Reply:
Either of Lance's suggestions sounds fine to me.

Reply:
the last suggestion is ok with me

Reply:
I'm good with Lance's suggestion too.

Reply:
I also like Lance's suggestion.

Reply:
OK, I've updated the document.